packages/utils/textElement.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  ExcalidrawElement,
+  NonDeletedExcalidrawElement,
+} from "../excalidraw/element/types";
+
+export const getContainerElement = (
+  element:
+    | (ExcalidrawElement & {
+        containerId: ExcalidrawElement["id"] | null;
+      })
+    | null,
+  sceneElements: readonly NonDeletedExcalidrawElement[],
+) => {
+  if (!element) {
+    return null;
+  }
+  if (element.containerId) {
+    return sceneElements.find((ele) => ele.id === element.containerId);
+  }
+  return null;
+};
```

Comment:
internally unused method -> bloat and great potential for future regressions

Reply:
This will be needed for future PRs hence I have decoupled it